cmake/jsoncpp.cmake

```diff
@@ -19,7 +19,19 @@ if(EMSCRIPTEN)
     # at the moment.
     set(JSONCPP_CXX_FLAGS -std=c++17)
 else()
-    set(JSONCPP_CXX_FLAGS ${CMAKE_CXX_FLAGS})
+    # jsoncpp uses implicit casts for comparing integer and
```

Comment:
Sounds like a good feature to activate without condition. That would also reduce the config complexity.

Reply:
Updated PR